pkg/k8shandler/visualization.go

```diff
@@ -188,7 +188,7 @@ func (clusterRequest *ClusterLoggingRequest) createOrUpdateKibanaDeployment() (e
 }
 
 func (clusterRequest *ClusterLoggingRequest) createOrUpdateOauthClient(oauthSecret string) (err error) {
-
+	logrus.Debug("createOrUpdateOauthClient for Kibana...")
```

Comment:
not sure this buys us anything, it should get into this function every time it processes unless a preceding one returns an error